gradle.properties

```diff
@@ -8,5 +8,5 @@ coroutines=1.3.3
 jacoco=0.8.5
 junit=4.13
 
-artifactPublish=2.3.0
+artifactPublish=3.0.0
```

Comment:
[nit] Since we need to have another PR to release version 3 (we probably need to improve the release management a bit later), shall we remove this out from this PR? 🙇 🙇 🙇